### ISSUE TITLE
Marktext-style editor

### ISSUE THREAD

**Issue by 2br-2b** [notemarks/notemarks#1]
Hello! Thanks for making this project open-source! The project is really well-done!

I wonder if it would be possible to implement a [Marktext](https://marktext.app/) or [Typora](https://typora.io/)-style editor for non-tech people? That way, they have an easier time editing without having to learn as much about how Markdown works.

Marktext is FOSS, Typora is not.

**Comment by bluenote10** [comment]
Yes, a WYSIWYG editor is definitely something I'd like to look into at some point. Marktext looks very interesting. It uses Vue, so it might be a bit of a challenge to integrate in React, but definitely worth a short.
